From 594d7a7c6b84fbd3ae7f8723ac456837af504acc Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 05 Jun 2019 13:17:26 +0300
Subject: [PATCH] [SW-0] gaudi: pll: rename gate switch w/a function

The hl_pll_hbw_gated_clk_sw() function is now used for switching all the
PLL dividers outputs, so rename it to hl_pll_gated_clk_sw().
The new name is more generic and better describes the functionality.

Change-Id: Ic24af421a637b06fb905d1622d8d5e2d377b1328
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index e09a795..087edf4 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -352,7 +352,7 @@
 	return ~gmask & 0xF;
 }
 
-static void hl_pll_hbw_gated_clk_sw(struct hl_pll_cfg *cfg)
+static void hl_pll_gated_clk_sw(struct hl_pll_cfg *cfg)
 {
 	int i;
 	u32 gmask = hl_pll_build_gate_mask(cfg);
@@ -395,6 +395,6 @@
 			break;
 		}
 
-		hl_pll_hbw_gated_clk_sw(&gaudi_pll_cfg[i]);
+		hl_pll_gated_clk_sw(&gaudi_pll_cfg[i]);
 	}
 }
